### PR TITLE
Feature/set up ci

### PR DESCRIPTION
**Pull request by creisle**
New Features
- Setup CI: Add tox/travis config

Improvements
- Reset URL to github repo
- Lint README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
-    - [Example of hawq table arguments with declarative syntax](#example-of-hawq-table-arguments-with-declarative-syntax)
-- [Using Partitions](#using-partitions)
+  - [Install (For developers)](#install-for-developers)
+  - [Run Tests](#run-tests)
+  - [Deploy (For developers)](#deploy-for-developers)
+- [Using in a SQLAlchemy project](#using-in-a-sqlalchemy-project)
+  - [How to incorporate sqlalchemy-hawq](#how-to-incorporate-sqlalchemy-hawq)
+  - [Hawq-specific table arguments](#hawq-specific-table-arguments)
+  - [Example of hawq table arguments with declarative syntax](#example-of-hawq-table-arguments-with-declarative-syntax)
+- [Using partitions](#using-partitions)
```
Comment (darryldee) [inline README.md:13]: The alignment of [ seems to be off here

File: setup.py [hunk 2]
```diff
@@ -30,7 +30,7 @@
 
 setup(
     name='sqlalchemy_hawq',
-    version='1.0.0',
+    version='1.0.1',
```
Comment (darryldee) [inline setup.py:33]: should the minor version number be bumped instead now that CI is setup which is a new feature?

Comment (creisle) [reply setup.py:33]: I went with patch since it is a feature that doesn't affect the package code at all (dev only)

File: README.md (hunk 1 above)
Comment (creisle) [reply README.md:13]: looks fine from what I can see, which part looks off?

Comment (darryldee) [reply README.md:13]: Not sure whether its my browser, but the opening [ on lines 9 and 13 seem under indented

Comment (creisle) [reply README.md:13]: Oh I see, they are just different levels of titles. They are supposed to look like that. Take a look at the README as it renders on the web page

https://github.com/bcgsc/sqlalchemy_hawq/tree/feature/set-up-ci

**Review by darryldee — APPROVED** [review]

**Pull request merged**